model/execution/DeliveryExecutionManagerService.php

```diff
@@ -259,8 +266,14 @@ public function updateDeliveryExtendedTime(DeliveryExecutionInterface $deliveryE
      * @param array $deliveryExecutions
      * @param int $seconds
      * @return array
+     * @throws common_Exception
+     * @throws common_exception_Error
+     * @throws common_exception_NotFound
+     * @throws common_ext_ExtensionException
+     * @throws InvalidServiceManagerException
+     * @throws QtiTestExtractionFailedException
      */
-    public function adjustTimers(array $deliveryExecutions, $seconds)
+    public function adjustTimers(array $deliveryExecutions, $seconds, array $reason = []): array
```

Comment:
Phpdoc updated with unhandled exceptions but is missing the new parameter

Reply:
fixed

Reply:
fixed